client/worker.go

```diff
@@ -29,8 +29,8 @@ import (
 	jaegerpb "github.com/jaegertracing/jaeger/model"
 	"go.opencensus.io/trace"
 
+	"github.com/signalfx/golib/v3/translator"
```

Comment:
This won't work as it'll still result in a circular dependency. Can we copy this one function (OCStatusCodeFromHTTP) to this package (client) and use it directly here?

Reply:
and here i thought i fixed it